From 2ba92e9996a0e854ef7d1cde3b1b636e9ede4346 Mon Sep 17 00:00:00 2001
From: Leigh Scott <leigh123linux@gmail.com>
Date: Wed, 27 May 2026 11:50:24 +0100
Subject: [PATCH 1/2] Fix UserWarning: pkg_resources is deprecated

---
 rfpkgdb2client/__init__.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rfpkgdb2client/__init__.py b/rfpkgdb2client/__init__.py
index 7956ac6..cbd98a9 100644
--- a/rfpkgdb2client/__init__.py
+++ b/rfpkgdb2client/__init__.py
@@ -19,7 +19,7 @@
 
 import getpass
 import logging
-import pkg_resources
+from importlib.metadata import version
 
 from six.moves import input, xrange
 
@@ -42,7 +42,7 @@ LOG = logging.getLogger("rfpkgdb2client")
 hand = NullHandler()
 LOG.addHandler(hand)
 
-__version__ = pkg_resources.get_distribution('rfpkgdb-cli').version
+__version__ = version('rfpkgdb-cli')
 PKGDB_URL = r'https://admin.rpmfusion.org/pkgdb'
 FAS_URL = r'https://admin.rpmfusion.org/accounts'
 BZ_URL = r'https://bugzilla.rpmfusion.org/xmlrpc.cgi'
-- 
2.54.0

