--- a/rfpkgdb2client/utils.py
+++ b/rfpkgdb2client/utils.py
@@ -26,7 +29,7 @@
 
 import requests
 import rpmfusion_cert
-import xmlrpclib
+import xmlrpc.client
 
 from bugzilla import Bugzilla
 from fedora.client import AccountSystem, AuthError
@@ -53,7 +56,7 @@
 
     try:
         BZCLIENT.logged_in
-    except xmlrpclib.Error:
+    except xmlrpc.client.Error:
         bz_login()
 
     return BZCLIENT
@@ -76,7 +79,7 @@
     print('To keep going, we need to authenticate against bugzilla'
           ' at {0}'.format(BZCLIENT.url))
 
-    username = raw_input("Bugzilla user: ")
+    username = input("Bugzilla user: ")
     password = getpass.getpass("Bugzilla password: ")
     BZCLIENT.login(username, password)
 
